css/notes.css

```diff
@@ -9,7 +9,6 @@
 }
 
 #app-navigation #note-add span {
-    display: none;
```

Comment:
Personally I think it adds clarity, and overall looks better. Having it look like a button like @korelstar said would also be nice, though was somewhat out of scope of what i was going for in this PR.